src/Builder/FormContractor.php

```diff
@@ -94,6 +94,9 @@ public function getFormBuilder($name, array $options = [])
 
     public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescription)
     {
+        // NEXT_MAJOR: Remove this line and update the function signature.
+        $formOptions = \func_get_args()[2] ?? [];
```

Comment:
Same like https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1142#discussion_r501724331

Reply:
@core23 The phpdoc is inherited from the interface getDefaultOptions.

I prefer to add the phpdoc in the interface in the SonataAdmin PR https://github.com/sonata-project/SonataAdminBundle/pull/6438/files